app/services/early_career_teachers/create.rb

```diff
@@ -5,31 +5,35 @@ class Create < BaseService
     include SchoolCohortDelegator
 
     def call
+      profile = nil
       ActiveRecord::Base.transaction do
         # Retain the original name if the user already exists
         user = User.find_or_create_by!(email: email) do |ect|
           ect.full_name = full_name
         end
         user.update!(full_name: full_name) unless user.participant_profiles&.active_record&.any? || user.npq_registered?
 
-        teacher_profile = TeacherProfile.find_or_create_by!(user: user) do |profile|
-          profile.school = school_cohort.school
+        teacher_profile = TeacherProfile.find_or_create_by!(user: user) do |teacher|
+          teacher.school = school_cohort.school
         end
 
-        ParticipantProfile::ECT.create!({
+        profile = ParticipantProfile::ECT.create!({
           teacher_profile: teacher_profile,
           schedule: Finance::Schedule::ECF.default,
           participant_identity: Identity::Create.call(user: user),
-        }.merge(ect_attributes)) do |profile|
-          ParticipantProfileState.create!(participant_profile: profile)
-
-          unless year_2020
-            ParticipantMailer.participant_added(participant_profile: profile).deliver_later
-            profile.update_column(:request_for_details_sent_at, Time.zone.now)
-            ParticipantDetailsReminderJob.schedule(profile)
-          end
-        end
+        }.merge(ect_attributes))
+
+        ParticipantProfileState.create!(participant_profile: profile)
+        Induction::Enrol.call(participant_profile: profile) if school_cohort.default_induction_programme.present?
```

Comment:
Can we have school cohorts without default induction programmes? How are profiles going to get induction records if we don't enrol on creation?

Reply:
Yes, there won't be a default induction programme if they choose "no ects" or "not yet know" options.
Currently I don't think you can add a participant until there is a programme to add them too, but we'll need to tease that out as we build the user journeys for the various scenarios